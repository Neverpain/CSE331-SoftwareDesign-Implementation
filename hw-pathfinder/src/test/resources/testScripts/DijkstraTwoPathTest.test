# Tests that Dijkstra's algorithm properly outputs the minimum-cost path
# between two nodes

CreateGraph g

AddNode g A
AddNode g B

AddEdge g A B 1.0
AddEdge g A B 10.0

FindPath g A B
