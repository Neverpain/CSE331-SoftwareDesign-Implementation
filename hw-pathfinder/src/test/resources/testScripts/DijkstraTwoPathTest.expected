# Tests that Dijkstra's algorithm properly outputs the minimum-cost path
# between two nodes

created graph g

added node A to g
added node B to g

added edge 1.000 from A to B in g
added edge 10.000 from A to B in g

path from A to B:
A to B with weight 1.000
total cost: 1.000